src/makefiles/cuda_64bit.mk

```diff
@@ -10,8 +10,8 @@ CXXFLAGS += -DHAVE_CUDA -I$(CUDATKDIR)/include -fPIC -pthread -isystem $(OPENFST
 CUDA_INCLUDE= -I$(CUDATKDIR)/include -I$(CUBROOT)
 CUDA_FLAGS = --machine 64 -DHAVE_CUDA \
              -ccbin $(CXX) -DKALDI_DOUBLEPRECISION=$(DOUBLE_PRECISION) \
-             -std=c++11 -DCUDA_API_PER_THREAD_DEFAULT_STREAM  -lineinfo \
-             --verbose -Xcompiler "$(CXXFLAGS)"
+             -std=c++14 -DCUDA_API_PER_THREAD_DEFAULT_STREAM  -lineinfo \
+             --verbose -Wno-deprecated-gpu-targets
```

Comment:
Just to make sure, are you removing `-Xcompiler "$(CXXFLAGS)"` intentionally here?

Reply:
Yes, they are no longer needed - I suspect we added the -Xcompiler back when we had more CPU code in .cu files. We've since moved the pure CPU code to .cc files to speed up compilation. 
It also prevents us from the config "cuda in c++14, rest of the code in c++11", as the c++11 flag in CXXFLAGS overrides the c++14 flag

Reply:
Yeah, just saw the very first comment. Actually, good riddance to this flag—I remember tweaking compilation flags and splitting what goes to nvcc and what should not. Thank you for this!